internal/parser/grammar.md

```diff
@@ -59,7 +59,7 @@ The parser directly implements the following grammar.
     MemberExpression               = DotExpression  | MemberBracketExpression
     DotExpression                  = "." identifer
     MemberBracketExpression        = "[" string "]" .
-    CallExpression                 = "(" ParameterList ")" .
```

Comment:
Even if they share the same implementation, parameter list and property list are different and so this should stay the same.

Reply:
Hmm, this doesn't seem right to me.  The parameter list is part of a function expression like `f = (x, y) => x * y`, but the property list is for calls like `f(x:5, y:10)`.  There are both `parsePropertyList` and `parseParameterList` methods in the parser, since they have different syntaxes.